snaps/qemu-op-tee-gadget/boot.cmd

```diff
@@ -0,0 +1,68 @@
+# Expects to be called with the following environment variables set:
+#
+#  devtype              e.g. mmc/scsi etc
+#  devnum               The device number of the given type
+#  distro_bootpart      The partition containing the boot files
+#                       (introduced in u-boot mainline 2016.01)
+#  prefix               Prefix within the boot partiion to the boot files
+#  kernel_addr_r        Address to load the kernel to
+#  ramdisk_addr_r       Address to load the initrd to.
+
+# these should probably be set somewhere else?
```

Comment:
I'd say it is ok to have these here, if the u-boot defaults for these values do not work. But the description of the script above needs to change now.